ezgui/src/canvas.rs

```diff
@@ -161,10 +182,24 @@ impl Canvas {
                 let displacement_unit_x = displacement_x / displacement_magnitude;
                 let displacement_unit_y = displacement_y / displacement_magnitude;
                 // Add displacement along each axis
-                self.cam_x += displacement_unit_x * PAN_SPEED;
-                self.cam_y += displacement_unit_y * PAN_SPEED;
+                pan_delta_x = displacement_unit_x * EDGE_PAN_SPEED;
```

Comment:
Will do!

Reply:
One thing I'm confused with here is that all of these seem like they engage in directly updating the camera. What makes edge panning different in this regard?

Reply:
UpdateType::Pan makes the control flow not pause and wait for input. We only want to influence control flow for autopanning, because we want to act like new events are happening while the cursor is idling at the edge. For pressing keys to pan or for dragging or anything else, we don't need to mess with the control flow, because the user is the one generating events